src/pymonzo/transactions/schemas.py

```diff
@@ -122,6 +122,41 @@ def __rich__(self) -> Table:
             return grid
 
 
+class MonzoTransactionCounterparty(BaseModel):
+    """API schema for a 'transaction counterparty' object.
+
+    Note:
+        This is undocumented in the Monzo API docs: https://docs.monzo.com/#transactions
+
+    Attributes:
+        account_number: The account number of the other party
+        name: The name of the other party
+        sort_code: The sort code of the other party
+        user_id: Monzo internal User ID of the other party
+    """
+
+    account_number: Optional[str] = None
+    name: Optional[str] = None
+    sort_code: Optional[str] = None
+    user_id: Optional[str] = None
```

Comment:
Should these be optional? I'd expect `counterparty` to be either set to `None` or have all these present, no?

Reply:
counterparty either comes through as:

`counterparty: {}`

or `counterparty: {
		account_number: "",
		name: "",
		sort_code: "",
		user_id: ""
	}`

and this is what worked for me... although happy to look again :)

Reply:
I think I've cleaned this up now...

Reply:
Well, that's not great on Monzo's side... OK, let me have a quick look into this and get back to you.